protocol/messenger_mailserver_cycle.go

```diff
@@ -648,6 +651,23 @@ func (m *Messenger) checkMailserverConnection() {
 	}
 }
 
+func (m *Messenger) EmitMailserverAvailable(peer string) {
+	for cancel, s := range m.mailserverCycle.availabilitySubscriptions {
+		select {
+		case s <- peer:
+		case <-cancel:
+			return
+		}
+	}
+}
+
+func (m *Messenger) SubscribeMailserverAvailable() (chan string, chan struct{}) {
+	c := make(chan string)
+	cancel := make(chan struct{})
+	m.mailserverCycle.availabilitySubscriptions[cancel] = c
+	return c, cancel
+}
```

Comment:
This is the other part of me trying to get a signal inside status-go that a mailserver is available.

Most likely better ways to do this. Appreciate input